src/MSBuild.UnitTests/XMake_Tests.cs

```diff
@@ -2369,6 +2369,7 @@ public void EndToEndWarnAsErrors(string switches, string errorCodes, string notE
             success.ShouldBe(expectedSuccess);
         }
 
+        // Test failed due to timeout. Windows-on-Core Full Framework.
```

Comment:
?

Reply:
I merged from main and the build failed on a test timeout. I needed a change to run the build again.

I can remove the comment

Reply:
Got it. Please do :)

Reply:
done.